src/DynamoCore/Models/RecordableCommands.cs

```diff
@@ -924,14 +924,28 @@ public class UpdateModelValueCommand : RecordableCommand
 
             #region Public Class Methods
 
-            public UpdateModelValueCommand(Guid modelGuid, string name, string value)
-                : this(new[] {modelGuid}, name, value)
+            /// <summary>
+            /// </summary>
+            /// <param name="modelGuid"></param>
+            /// <param name="name"></param>
+            /// <param name="value"></param>
+            /// <param name="workspaceModel">If workspace model is null, this command will be sent to the current workspace</param>
+            public UpdateModelValueCommand(Guid modelGuid, string name, string value, WorkspaceModel workspaceModel = null)
+                : this(new[] {modelGuid}, name, value, workspaceModel)
```

Comment:
Yep I think I got that, but still the original comment above applies: `workspaceGuid` can be added before `modelGuid`, and then remove that `WorkspaceModel` argument.